modules/web/src/de/diedavids/cuba/dataimport/web/importwizard/ImportWithImportConfigurationWizard.groovy

```diff
@@ -159,7 +159,7 @@ class ImportWithImportConfigurationWizard extends AbstractEditor<ImportConfigura
             log.error(e.message, e)
 
             startImport.enabled = false
-            showNotification(formatMessage('uploadFileDoesNotMatchExpectedStructure'), Frame.NotificationType.ERROR)
```

Comment:
note sure about this change. This way other people cannot define translations for them, right?

Perhaps alternatively you can you like `formatMessage('uploadFileDoesNotMatchExpectedStructure', e.getResult())` where `getResult` is the method of the exception which contains the validation result.

Reply:
I generated scripts using Cuba features, installed docker images from mysql and postgress as you suggested at create-databases.md.
Translation changes are coming soon